ui/inspectors/system-section.reel/updates/updates.reel/updates.js

```diff
@@ -71,7 +71,12 @@ exports.Updates = Component.specialize(/** @lends Updates# */ {
 
     handleCheckDownloadAction: {
         value: function() {
-            this._updateService.checkAndDownload();
+            var self = this;
+            this._updateService.checkAndDownload().then(function() {
+                return self._updateService.getInfo();
+            }).then(function(info) {
+                self.info = info;
```

Comment:
`info` should be a `UpdateService` property that would be refreshed by the service without the inspector having to bother with that.